### ISSUE TITLE
NodeB v1.8.2 - Can't Upload Files using Redactor

### ISSUE THREAD

**Issue by TroikaTronix** [NodeBB/NodeBB#6486]
- **NodeBB version:1.8.2** 
- **NodeBB git hash:???** 
- **Database type:** mongo
- **Database version: v3.2.19** 
- **Exact steps to cause this issue:**
N.B. If Composer is enabled as the editor, this issue does not happen. It is specific to Redactor.
1. Enable Redactor as your editor
2. Start a new topic or hit reply to an existing one
3. Attempt to drop a file into the editor, or to click the "File" button to upload. In either case, the progress bar appears at the top but no file appears.
4. Workaround: if you click the "File" button and enter text next to the "Name (Optional)" then the link _will_ appear as expected. There is no workaround for for drag and drop.
5. I tested this on a fresh NodeBB site and it is not our configuration. The same problem exists on the fresh install.
- **What you expected:**
The file to appear with a link so that I can download.
- **What happened instead:**
Nothing appeared at at.

**Comment by julianlam** [comment]
Can you report your redactor version? I am unable to reproduce.

Also I see this error when loading the page `5:125 Uncaught ReferenceError: $ is not defined`, which I don't know for certain if it is responsible, but could be.

**Comment by TroikaTronix** [comment]
Redactor v3.2.0 - NodeBB says it's the latest.

**Comment by julianlam** [comment]
Indeed... although I couldn't repro on my dev environment. Will give a clean deploy a try.

**Comment by julianlam** [comment]
Yep, definitely can reproduce now on a clean environment, thanks for reporting. You can also delete the test instance on our hosting now :laughing:

**Comment by TroikaTronix** [comment]
Thanks @julianlam - much appreciated.
